### ISSUE TITLE
Web interface: After adding a queue via command line the frontend does not show all queues

### ISSUE THREAD

**Issue by tillkamppeter** [michaelrsweet/pappl#155]
In the PostScript Printer Application (this is the only working PAPPL-based Printer Application with PAPPL's standard main loop and so PAPPL's standard command line client interface) I have set up 2 print queues with the names "test" and "test2", which are displayed in alphabetical order on the front page of the web interface (First screen shot).

Then I add a third print queue named "test1", via command line:
```
./ps-printer-app add -d test1 -v 'usb://HP/HP%20OfficeJet%20Pro%208730?serial=CN783F60W1' -m hp--officejet-pro-8730-user-added--en
```
The command is accepted and exits with 0, so the queue should be created, and looking into the web interface again, displaying the front page again (Second screen shot), the queue actually appears, also in the correct position, right after "test", but the third queue, "test2" has disappeared.

Actually "test2" is not lost, as when I display the front page again, without doing anything inbetween, all the three queues show up (Third screen shot).

Here is the log (boring things as PPD repo indexing and get-printer-attributes responses to CUPS/cups-browsed skipped): 
[ps-printer-app-log-cmdline-add-20210221.txt](https://github.com/michaelrsweet/pappl/files/6018679/ps-printer-app-log-cmdline-add-20210221.txt)

First screen shot:
![Screenshot from 2021-02-21 22-41-33](https://user-images.githubusercontent.com/1244825/108640219-b70f0d00-7498-11eb-8676-1d58c7fcbfb8.png)

Second screen shot:
![Screenshot from 2021-02-21 22-43-02](https://user-images.githubusercontent.com/1244825/108640233-c726ec80-7498-11eb-9e9f-3904bb008ae1.png)

Third screen shot:
![Screenshot from 2021-02-21 22-43-13](https://user-images.githubusercontent.com/1244825/108640253-d312ae80-7498-11eb-84e0-e791c1ccda3a.png)

**Comment by michaelrsweet** [comment]
Found some more threading issues, and I think this should resolve things once and for all!

[master 1d55a26] Fix some more threading issues (Issue #155, Issue #162)

[v1.0.x b9cdc23] Fix some more threading issues (Issue #155, Issue #162)

**Comment by tillkamppeter** [comment]
It works correctly now. Thank you very much.
